README.md

```diff
@@ -1,19 +1,19 @@
 # Kentico Cloud sample React single-page application
+
 [![Build Status](https://api.travis-ci.com/Kentico/cloud-sample-app-react.svg?branch=master)](https://travis-ci.com/Kentico/cloud-sample-app-react)
 [![Live Demo](https://img.shields.io/badge/live-demo-brightgreen.svg)](https://kentico-sample-app-react.surge.sh)
 [![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-ASK%20NOW-FE7A16.svg?logo=stackoverflow&logoColor=white)](https://stackoverflow.com/tags/kentico-cloud)
 
 This is a sample website written in JavaScript utilizing the Kentico Cloud Delivery API to manage and retrieve content. You can register your account for free at <https://app.kenticocloud.com>. For a brief walkthrough, read [Running the React sample app](https://developer.kenticocloud.com/v1/docs/running-react-sample-app) on our Developer Hub.
 
-
 ## Application setup
 
-1. Install the latest version of NodeJS and npm. You can download both at <https://nodejs.org/en/download/>.
-2. Clone the sample application repository.
-3. Navigate to the root folder of the application in the command line.
-4. Type `npm install` to install required npm packages.
-5. Type `npm start` to start a development server.
-6. The application opens in your browser at <http://localhost:3000>.
+1.  Install the latest version of NodeJS and npm. You can download both at <https://nodejs.org/en/download/>.
```

Comment:
Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030

Reply:
Oh whoops